ui/desktop/app/styles/app.scss

```diff
@@ -292,3 +292,38 @@
   margin: sizing.rems(l) 0 0 0;
   text-align: center;
 }
+
+.credentials-card {
+  border: sizing.rems(xxxxs) solid var(--ui-gray-subtler-3);
+  display: flex;
+  flex-direction: column;
+  margin-bottom: sizing.rems(m);
+
+  &:last-child {
+    margin-bottom: 0;
+  }
+
+  .credentials-card-header {
```

Comment:
The naming is not definitive. I call it `credentials-card` because we already have a `card` in rose. Should I call it just card? should I call it something else?